context/app/static/js/components/Header/HeaderContent/style.ts

```diff
@@ -0,0 +1,27 @@
+import { styled } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+import Divider from '@mui/material/Divider';
+import SvgIcon from '@mui/material/SvgIcon';
+
+const Spacer = styled('div')({
```

Comment:
I imagine these are in-place conversions, but we could replace the divs here with `Box`/`Stack`.

Reply:
I removed some of the other unused styles as well 👍🏻